pkgs/by-name/ra/rabbitmq-server/package.nix

```diff
@@ -106,6 +107,19 @@ stdenv.mkDerivation rec {
     rm $out/INSTALL
   '';
 
+  # Can not use versionCheckHook since that doesn't allow for setting environment variables
+  # which is necessary since Erlang needs a $HOME for the Cookie.
```

Comment:
`testers.testVersion` can be used with custom env variables. Something along the lines of: https://github.com/NixOS/nixpkgs/blob/b10e42649fd8fd7aff02e83c7853c80eefe92644/pkgs/by-name/fe/feather/package.nix#L75-L80

Reply:
> I'd advocate for the hook

Fine by me, but I'd like for someone else to chime in as well.